From a61b529003472e3827e269835399978f68d0925c Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Mon, 28 Dec 2015 14:24:11 +0800
Subject: [PATCH 0806/5242] MLK-12087 ARM: dts: imx6ul-9x9-evk.dts: add
 dual-role support for otg port

commit  2f83eec161400ea74435d56985996468b724722e from
https://source.codeaurora.org/external/imx/linux-imx.git

Add dual-role support for imx6ul-9x9 evk board, the full otg function
is not supported at this board due to the vbus can't be controlled
by software.

Signed-off-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul-9x9-evk.dts |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ul-9x9-evk.dts b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
index 08088cf..00623b0 100644
--- a/arch/arm/boot/dts/imx6ul-9x9-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
@@ -742,7 +742,10 @@
 };
 
 &usbotg1 {
-	dr_mode = "peripheral";
+	dr_mode = "otg";
+	srp-disable;
+	hnp-disable;
+	adp-disable;
 	status = "okay";
 };
 
-- 
1.7.9.5

